packages/EasyTest/src/InvalidDataMaker/InvalidDataMaker.php

```diff
@@ -6,6 +6,10 @@
 
 use Carbon\Carbon;
 use Doctrine\Common\Collections\ArrayCollection;
+use EonX\EasyCore\Bridge\Symfony\Validator\Constraints\Alphanumeric;
```

Comment:
We need to add `EasyCore` to the dependencies in `composer.json`.

Reply:
Done.